test/license_test.dart

```diff
@@ -46,7 +46,8 @@ void main() {
       await expectFile('test/licenses/bsd_2_clause.txt', ['BSD-2-Clause']);
       await expectFile(
           'test/licenses/bsd_2_clause_in_comments.txt', ['BSD-2-Clause']);
-      await expectFile('test/licenses/bsd_3_clause.txt', ['unknown']);
+      await expectFile(
+          'test/licenses/bsd_3_clause.txt', ['BSD-3-Clause-Clear']);
```

Comment:
Why is there a "BSD" inserted in the first line?

Maybe, for debugging purposes we should make a utility to dump the diff... maybe as HTML with colored spans or something?
So that it's easy to see what is inserted and removed... so we can reason about whether it's a mistake in the diffing or if we should make our induces phrase checks more precise (we can certainly make tricks where if the thing inserted is `apache.org` maybe it's not bad, but inserting `apache` is bad).